### PR TITLE
Inclusion of the infamous Floyd's cycle finding algorithm as another approach (previous one left unchanged) to detect/check for cycles/loops in a linkedlist in is_cyclic function.

### PR DESCRIPTION
**Pull request by Anirban166**
(Put an `X` inside the `[ ]` to denote check mark `[X]`.)

- [ X] **If creating a new file :**
  - [X ] added links to it in the README files ?
  - [ ] included tests with it ?
  - [ ] added description (overview of algorithm, time and space complexity, and possible edge case) in docstrings ?

- [ ] **if done some changes :**
  - [X ] wrote short description in the PR explaining what the changes do ?
  - [ ] Fixes #[issue number] if related to any issue

- [ ] **other**

### CODE REVIEW

File: algorithms/linkedlist/is_cyclic.py [hunk 1]
```diff
@@ -25,3 +25,56 @@ def is_cyclic(head):
...
+            else:
+                 return False
+            slow = slow.next  
+            if slow == fast:
+                print("--slow",slow.data, "fast",fast.data) 
+                return True    
+        return False
+
+# Create a LinkedList:
+linkedList = LinkedList()
```
Comment (keon) [inline algorithms/linkedlist/is_cyclic.py]: please add these to unittests

File: algorithms/linkedlist/is_cyclic.py [hunk 2]
```diff
@@ -25,3 +25,56 @@ def is_cyclic(head):
...
+    def strnode (self):
+        print(self.data)
+
+# LinkedList class:
+class LinkedList:
+    def __init__(self):
+        self.numnodes = 0
+        self.head = None
+
+    def insertLast(self, data):
```
Comment (keon) [inline algorithms/linkedlist/is_cyclic.py]: please use consistent naming convention (snake_case)
